composer/workflows/airflow_db_cleanup.py

```diff
@@ -192,6 +192,21 @@
 except Exception as e:
     logging.error(e)
 
+# Check for RenderedTaskInstanceFields model
+if AIRFLOW_VERSION < ["2", "4", "0"]:
```

Comment:
Hello, in airflow version > 2.4.0 the table still exists.

Why do you apply a filter?